tokio/src/future/mod.rs

```diff
@@ -22,3 +22,12 @@ cfg_sync! {
     mod block_on;
     pub(crate) use block_on::block_on;
 }
+
+cfg_trace! {
+    mod trace;
+    pub(crate) use trace::InstrumentedFuture as Future;
```

Comment:
This rename was mostly for convenience in all the other modules. But it could be that people find it too confusing, in which case I'll ditch the rename and just make the name change in all the other places.

Reply:
Does this change the public API of spawn methods to use a different trait, or is it only internal? I can't really tell.

Reply:
No it doesn't. And I checked in a [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=16b62490c64cf7fce9f9702cd118a854), since the trait is `pub(crate)` the compiler will error if the bounds were used on a `pub` function.